source/fundamentals/crud/read-operations/distinct.txt

```diff
@@ -21,20 +21,23 @@ across a collection.
 Sample Documents
 ~~~~~~~~~~~~~~~~
 
-Consider a collection called ``restaurants`` that contains documents with restaurant data.
-The examples on this page use the following ``restaurants`` documents to demonstrate the
-usage of ``distinct()``:
+Use the following code snippet to insert documents with restaurant
+data to the ``myDB.restaurants`` collection to follow along with the
+examples in this guide:
```

Comment:
S: Same suggestion as above for a minor improvement (this sentence isn't as a confusing)

```suggestion
To follow along with the examples in this guide,
use the following code snippet to insert documents with restaurant
data to the ``myDB.restaurants`` collection:
```

Reply:
changed in all places where this intro is used.